### ISSUE TITLE
How to convert the aimet pd model and model encodings file to dlc file 

### ISSUE THREAD

**Issue by KONE-2021** [quic/aimet#362]
When I convert the aimet pd model and model encodings file to dlc file ，an error occurred:
tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'QcQuantize' in binary running on . Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
How can i fix this error?

**Comment by quic-akhobare** [comment]
Hi @KONE-2021 

You should remove the quantization wrappers from the model before trying to convert it to DLC. Please see the QuantizationSimModel.export() API
https://quic.github.io/aimet-pages/AimetDocs/api_docs/torch_quantsim.html#aimet_torch.quantsim.QuantizationSimModel.export

**Comment by ibei** [comment]
> Hi @KONE-2021
> 
> You should remove the quantization wrappers from the model before trying to convert it to DLC. Please see the QuantizationSimModel.export() API
> https://quic.github.io/aimet-pages/AimetDocs/api_docs/torch_quantsim.html#aimet_torch.quantsim.QuantizationSimModel.export


> When I convert the aimet pd model and model encodings file to dlc file ，an error occurred:
> tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'QcQuantize' in binary running on . Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
> How can i fix this error?

Did you use snpe API to do this?

**Comment by quic-akhobare** [comment]
No update in a while. AIMET export will natively export PyTorch models to ONNX.

Closing this ticket. If there is a follow-up question please re-open or create a new issue.
